crm_rma_location_rma/models/stock_warehouse.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright 2013 Camptocamp
+#    Copyright 2015 Vauxoo
+#    Author: Yanina Aular
+#            Osval Reyes
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import _, api, fields, models
+
+
+class StockWarehouse(models.Model):
+
+    _inherit = "stock.warehouse"
+
+    lot_rma_id = fields.Many2one('stock.location', 'RMA Location')
+    rma_out_type_id = fields.Many2one('stock.picking.type', 'RMA Out Type')
+    rma_in_type_id = fields.Many2one('stock.picking.type', 'RMA In Type')
+    rma_int_type_id = fields.Many2one('stock.picking.type',
+                                      'RMA Internal Type')
+
+    @api.model
+    def create_sequences_picking_types(self, warehouse):
+        """
+        Takes care of create picking types for internal,
+        incoming and outgoing RMA
+        """
+        seq_obj = self.env['ir.sequence']
```

Comment:
OK, I see, maybe a loop passing some arguments can reduce a bit the code, but it's not critical.